docs/application/native/guides/app-management/sticker.md

```diff
@@ -0,0 +1,431 @@
+# Sticker
+
+
+The Sticker feature provides the functions to manage and retrieve sticker information. The Sticker application is divided into the [Sticker provider application](#provider_application) that creates the sticker and the [Sticker consumer application](#consumer_application) that uses the sticker created by the Sticker provider. The Sticker provider application can store sticker details such as URI, keyword, and group name to the Sticker database. The Sticker consumer application can retrieve sticker information using the Sticker API.
+
+The main features of the Sticker API include:
+
+- Preparing the Sticker service for use
+
+  You can connect the Sticker daemon to be able to operate the Sticker service.
+
+- Sticker data handle
+
+  You can use the Sticker data handle to [store the sticker information](#set_data_handle) and to [get the sticker information](#get_data_handle) stored in it.
+
+- [Inserting the sticker information](#insert_sticker)
+
+  - You can store sticker information using the Sticker API at application runtime, and [use JSON file](#set_json) to store many sticker information at once.
```

Comment:
Please add fix articles.
Can we remove "many" before "sticker information" and rephrase to:
You can store the sticker information using the Sticker API at application runtime, and [use the JSON file](#set_json) to store the sticker information at a time.

Please verify.

Reply:
Fixed.